jsa/features/people.feature

```diff
@@ -12,13 +12,13 @@ Feature: People
     And I should see the "People" heading in the "First sidebar" region
     And I should see "This is your Why I Teach block" in the "First sidebar" region
 
-  @content @live
+  @content @deploy
   Scenario: Faculty node
     Given I am on "people/faculty"
     And I click "Jacob Smith" in the "Content Body" region
     Then I should see "Professor of English" in the "Content Body" region
 
-  @content @live
+  @content @deploy
```

Comment:
If the site owner removes Jacob Smith then this would succeed when deployed but fail when the site is live. Is this what is expected?